src/Git/Git.php

```diff
@@ -252,4 +252,18 @@ protected function push($gitRoot)
     {
         GitProcess::create($gitRoot)->push();
     }
+
+    /**
+     * Shell escape string for use in git commands.
+     *
+     * @param string $string
+     * @return string
+     */
+    protected function shellEscape(string $string)
+    {
+        $string = str_replace('"', '', $string);
+        $string = str_replace("'", '', $string);
```

Comment:
This `shellEscape` function is now used to escape Git user names (line 243). It strips out single quotes. What happens when a user's name legitimately has single quotes in them, such as e.g. "A'idah"?

Reply:
Valid. 👍

Reply:
We're leaving this fix as is for now. It's a bit naive as it'll strip out the quotes from cases like O'Brien like you said, but that's cosmetic - it'll only be in the git log.